src/view/web/modal/ModalContent.js

```diff
@@ -0,0 +1,74 @@
+export default function ModalContent() {
+  const main = document.createElement('main');
+  // 닫기 버튼
+  const closeButtonSection = document.createElement('section');
+  closeButtonSection.setAttribute('id', 'close-btn-container');
+
+  const closeButton = document.createElement('button');
+  closeButton.setAttribute('id', 'close-btn');
+
+  const closeBtnText = document.createElement('span');
+  closeBtnText.setAttribute('id', 'close-btn-text');
+  closeBtnText.innerText = 'X';
+
+  closeButton.appendChild(closeBtnText);
+  closeButtonSection.appendChild(closeButton);
+
+  // 당첨 통계 제목
+  const statisticsTitleContainer = document.createElement('section');
+  statisticsTitleContainer.setAttribute('id', 'statistics-title-container');
+
+  const h1 = document.createElement('h1');
+  h1.innerText = '🏆 당첨 통계 🏆';
```

Comment:
요게 정말 `h1` 태그여야할까요? 👀

Reply:
처음에는 하나의 페이지당 하나의 h1 tag 가 있는게 좋을 것 같아서 드린 리뷰였는데,
저도 다시 생각해보니 모달 내의 타이틀이어서 h1 tag 로 가는게 옳은 방향인 것 같아요! 

찾아보니 MUI 나 bootstrap 에서도 접근성의 이유로 모달 title 을 h2에서 h1 tag로 변경했네요!
[[Dialog] DialogTitle should use h1 instead of h2](https://github.com/mui/material-ui/issues/34250)